server/src/internalClusterTest/java/org/opensearch/indices/replication/SegmentReplicationRelocationIT.java

```diff
@@ -55,6 +56,7 @@ private void createIndex(int replicaCount) {
      * This test verifies happy path when primary shard is relocated newly added node (target) in the cluster. Before
      * relocation and after relocation documents are indexed and documents are verified
      */
+    @TestLogging(reason = "Getting trace logs from replication package", value = "org.opensearch.indices.replication:TRACE")
```

Comment:
1. I suspect this problem appears due to the delay in routing changes propagation to old primary (and thus replication event not cancelled). It will be worth adding loggs for Allocation and IndexShard package. 
 2. Since we are only cleaning up and enabling logging as part of this PR, we can update the description.